scripts/paper/generate_stwo_shared_table_reuse_figure.py

```
@@ -0,0 +1,393 @@
+  <text x="{WIDTH/2:.1f}" y="{HEIGHT - 24}" text-anchor="middle"
+        font-family="STIX Two Text, Georgia, serif" font-size="20" fill="#6B7280">
+    {escape(footnote_2)}
+  </text>
+</svg>
+'''
+
+
+def write_optional_rasters(svg_path: Path, png_path: Path, pdf_path: Path) -> None:
+    tmp_png = png_path.with_suffix(png_path.suffix + ".tmp")
+    try:
+        rsvg = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)